index.d.ts

```diff
@@ -0,0 +1,30 @@
+import { Accepts } from "accepts"
+import * as fastify from "fastify"
+import { Server, IncomingMessage, ServerResponse } from "http"
+
+declare module "fastify" {
+  interface FastifyRequest<HttpRequest> extends Accepts {
+    accepts(): Accepts
+  }
+
+  interface FastifyReply<HttpResponse> {
```

Comment:
Can you do the same thing you did to FastifyRequest?

Reply:
I'm afraid it won't work, as FastifyReply has prefixes on all properties, starting with `request`